app/assets/javascripts/admin/views/edition-form.js

```diff
@@ -62,5 +63,38 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  EditionForm.prototype.setupSpeechSubtypeEventListeners = function () {
+    var form = this.module
+
+    var select = form.querySelector('#edition_speech_type_id')
+
+    if (!select) { return }
+
+    var deliveredByLabel = form.querySelector('#edition_role_appointment .govuk-fieldset__heading')
+    var hasProfileRadioLabel = form.querySelector('#edition_role_appointment label[for="edition_role_appointment_speaker_on_govuk"]')
+    var noProfileRadioLabel = form.querySelector('#edition_role_appointment label[for="edition_role_appointment_speaker_not_on_govuk"]')
+    var deliveredOnLabel = form.querySelector('#edition_delivered_on .govuk-fieldset__legend')
+    var locationDiv = form.querySelector('.js-app-view-edition-form__speech-location')
+    var locationInput = locationDiv.querySelector('input[name="edition[location]"]')
+    var authoredArticleId = '6'
+
+    select.addEventListener('change', function (event) {
+      if (event.currentTarget.value === authoredArticleId) {
+        locationDiv.classList.add('govuk-visually-hidden')
```

Comment:
`'govuk-visually-hidden'` will only remove this for non-screenreader users, I think we want it to be hidden for everyone, so we should probably use the `hidden` property.

```suggestion
        locationDiv.hidden = true
```

Reply:
When looking at this i noticed for the locale switcher added a class with `display: none`

<img width="375" alt="image" src="https://user-images.githubusercontent.com/42515961/212881651-513bb48a-2d21-4b34-948f-d1892ec4b704.png">

So i've just piggy backed off of that so it'll apply correctly on page load. Does that sound reasonable?